tests/Mock/IgnWfsMockClient.php

```diff
@@ -26,42 +26,85 @@ private function handleRequests(string $method, string $url, array $options): Mo
 
     private function getWfsMock($options): MockResponse
     {
-        if (str_contains($options['query']['cql_filter'], "strStripAccents(strReplace(nom_minuscule, '-', ' ', true))=strStripAccents(strReplace('rue saint victor', '-', ' ', true))")) {
-            $body = [
-                'features' => [
-                    [
-                        'geometry' => [
-                            'type' => 'MultiLineString',
-                            'coordinates' => [
-                                [
-                                    [3.06773711, 50.65302845],
-                                    [3.06770772, 50.65320974],
-                                    [3.0676489, 50.65356155],
-                                    [3.06761116, 50.65383438],
-                                    [3.06756215, 50.65413324],
-                                    [3.06756371, 50.65424093],
+        if ($options['query']['TYPENAME'] === 'BDTOPO_V3:route_numerotee_ou_nommee') {
+            if (str_contains($options['query']['cql_filter'], "strStartsWith(numero, 'D32')=true AND gestionnaire='Ardennes' AND type_de_route='Départementale'")) {
+                $body = [
+                    'features' => [
+                        [
+                            'properties' => [
+                                'numero' => 'D322',
+                            ],
+                            'geometry' => [
+                                'type' => 'MultiLineString',
+                                'coordinates' => [
+                                    [
+                                        [4.66349228, 49.8207711],
+                                        [4.66356107, 49.82070816],
+                                        [4.6636232, 49.8206543],
+                                        [4.66372513, 49.82058551],
+                                        [4.66385317, 49.82050828],
+                                        [4.66399657, 49.82043354],
+                                        [4.66415639, 49.82035139],
+                                        [4.6643028, 49.82028379],
+                                        [4.66443686, 49.82022086],
+                                        [4.66459579, 49.82015399],
+                                        [4.6647601, 49.82008166],
+                                    ],
                                 ],
                             ],
```

Comment:
(Détail) Je ne sais pas si tu (Léa / Mathieu) as dû manuellement transformer le JSON en array PHP, mais pour se simplifier la vie on peut utiliser la string JSON un peu partout

```suggestion
                            'geometry' => json_decode('{"type":"MultiLineString","coordinates":[[[4.66349228,49.8207711],[4.66356107,49.82070816],[4.6636232,49.8206543],[4.66372513,49.82058551],[4.66385317,49.82050828],[4.66399657,49.82043354],[4.66415639,49.82035139],[4.6643028,49.82028379],[4.66443686,49.82022086],[4.66459579,49.82015399],[4.6647601,49.82008166]]]}', associative: true),
```